src/codeu/chat/common/User.java

```diff
@@ -26,13 +28,18 @@
 import com.fasterxml.jackson.annotation.JsonIgnore;
 import com.fasterxml.jackson.annotation.JsonIgnoreProperties;
 
+  public long updateTime = Time.now().inMs();
+  public HashMap<Uuid, Integer> conversationInterests = new HashMap<>();
```

Comment:
Are these public for a specific reason? Maybe add comments explaining what these data structures hold.